modules/oe_webtools_globan/src/Form/WebtoolsGlobanSettingsForm.php

```diff
@@ -118,6 +129,7 @@ public function submitForm(array &$form, FormStateInterface $form_state): void {
       ->set('background_theme', $form_state->getValue('background_theme'))
       ->set('display_eu_institutions_links', (bool) $form_state->getValue('display_eu_institutions_links'))
       ->set('override_page_lang', $form_state->getValue('override_page_lang'))
+      ->set('sticky', $form_state->getValue('sticky'))
```

Comment:
It looks like missed casting to the boolean data type.
Config schema accepts boolean.